ansible/templates/minigraph_dpg_asic.j2

```diff
@@ -1,6 +1,11 @@
 {# Note max of 10 Backend Portchannel from one asic #}
 {% macro port_channel_id(asic_idx, neigh_asic_idx) -%}
+{% if card_type is defined and card_type == 'supervisor' %}
+{# Note avoid PortChannel00 #}
```

Comment:
can we have separate PR and we can remove supervisor check.

Reply:
Removed this change from this PR and created separate PR: https://github.com/Azure/sonic-mgmt/pull/5705